tox.ini

```diff
@@ -300,7 +300,6 @@ doctest_optionflags = NORMALIZE_WHITESPACE IGNORE_EXCEPTION_DETAIL ELLIPSIS
 filterwarnings =
     ignore:distutils Version classes are deprecated:DeprecationWarning
```

Comment:
Unsure of why this warning is still coming up. Maybe from an imported package? I don't think we're directly using any of the `pkg_resources` versions now.